packages/site/src/pages/index.tsx

```diff
@@ -459,7 +481,6 @@ const Index = () => {
             <CustomMessageButton
               $text="Grant Permission"
               onClick={handleGrantPermissions}
-              disabled={isWorking}
```

Comment:
I noticed the "Grant Permission" button stays active while processing. Do we want it disabled, like the "Redeem Permission" button?

```typescript
disabled={pendingPermissionRequests.size > 0} // Add this
```

Reply:
Specifically wanted it to stay active. So you can try to click grant permission and see it gets rejected until the existing permission is processed. Forgot to update the description on how to test.